src/workerd/api/sockets.h

```diff
@@ -104,8 +104,8 @@ class Socket: public jsg::Object {
   // `startTls`.
   kj::String domain;
   // The domain/ip this socket is connected to. Used for startTls.
-  int port;
-  // The port this socket is connected to. Used for nicer errors.
+  bool isFetchablePort;
```

Comment:
Not sure if `fetchable` is the right adjective here. Any port is potentially fetchable. Perhaps `isDefaultFetchPort`?